bin/DashboardNotifier.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import division
+import stomp
+import json
+import traceback
+import os
+import datetime
+import logging 
+from multiprocessing import Process
+
+def produce(file_path, logging, conn):
+    """
+    """
+    json_data=open(file_path)
+    message = json.load(json_data)
+    logging.debug("Producing...%s" % message)
+    try:
+        messageDict = json.dumps(message)
+        conn.send(messageDict, destination=authParams['MSG_QUEUE'] )
+    except Exception as ex:
+        msg = "Error contacting Message Broker"
+        msg += str(ex)
+        msg += str(traceback.format_exc())
+        logging.debug(msg)
```

Comment:
You can also do logging.exception("Error contacting Message Broker"). It gives you the same result of the 4 lines above